opslevel.yml

```diff
@@ -15,7 +15,7 @@ service:
   description: >
     Devspace telemetry plugin
   lifecycle: public_ga
-  tier: tier_0
```

Comment:
we may need to add `tier: 2` under opslevel in https://github.com/getoutreach/devtel/blob/main/service.yaml#L17. otherwise, re-stencil will revert it